docs/releasenotes/3.3.0.rst

```diff
@@ -56,6 +56,25 @@ Our internal reports structure was refactored to make it easier to maintain and
 each report is stored in a separate file in Robocop project. From the user's perspective, reports should work
 the same as before.
 
+Severity threshold print value (#791)
+-------------------------------------
+
+Rules with severity threshold should now print the message with the
```

Comment:
I will rewrite release notes later, because I see few typos or sentences that could be worded better.

Reply:
Sure. I will review this part later then.

Reply:
Done